src/main/java/com/powsybl/openloadflow/OpenLoadFlowProvider.java

```diff
@@ -289,20 +290,29 @@ private LoadFlowResult runDc(Network network, LoadFlowParameters parameters, Ope
 
     @Override
     public CompletableFuture<LoadFlowResult> run(Network network, ComputationManager computationManager, String workingVariantId, LoadFlowParameters parameters) {
+        return run(network, computationManager, workingVariantId, parameters, Reporter.NO_OP);
+    }
+
+    @Override
+    public CompletableFuture<LoadFlowResult> run(Network network, ComputationManager computationManager, String workingVariantId, LoadFlowParameters parameters, Reporter reporter) {
         Objects.requireNonNull(workingVariantId);
         Objects.requireNonNull(parameters);
 
         LOGGER.info("Version: {}", new PowsyblOpenLoadFlowVersion());
 
         OpenLoadFlowParameters parametersExt = getParametersExt(parameters);
 
+        Reporter lfReporter = reporter.createSubReporter("loadFlow", "Load flow on network ${networkId}",
```

Comment:
What is the aim of the word "loadflow"?

Reply:
It is a key identifying the task, this allows to have multilingual support by having several versions (one for each language) of the taskname.